src/main/java/com/google/api/codegen/DiscoveryFragmentGeneratorApi.java

```diff
@@ -107,15 +110,14 @@ protected void process() throws Exception {
       return;
     }
 
-    String overridesFilename = options.get(OVERRIDES_FILE);
-    JsonNode overridesJson = null;
-    if (!Strings.isNullOrEmpty(overridesFilename)) {
+    String[] filenames = options.get(OVERRIDE_FILES).split(",");
+    List<JsonNode> overrides = new ArrayList<>();
+    for (String filename : filenames) {
       try {
         BufferedReader reader =
-            com.google.common.io.Files.newReader(
-                new File(overridesFilename), Charset.forName("UTF8"));
+            com.google.common.io.Files.newReader(new File(filename), Charset.forName("UTF8"));
         ObjectMapper mapper = new ObjectMapper();
-        overridesJson = mapper.readTree(reader);
+        overrides.add(mapper.readTree(reader));
```

Comment:
An exception will be raised when parsing into the `SampleConfig` class. Since it's an `AutoValue`, it'll complain if any types are wrong, or if the expected structure isn't matched.